ironfish/src/fileStores/config.ts

```diff
@@ -377,7 +377,7 @@ export class Config extends KeyStore<ConfigOptions> {
       feeEstimatorPercentileLow: 10,
       feeEstimatorPercentileMedium: 20,
       feeEstimatorPercentileHigh: 30,
-      networkId: 2,
+      networkId: 1,
```

Comment:
Should we default to the dev network since we don't have mainnet running right now?